test/e2e/metrics_e2e_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build !bare
```

Comment:
I'm not familiar with this.  What is this for?

Reply:
oh weird - I'm not sure what happened there? it looks like my IDE automatically added this so I'll have to dive more into this later.

Reply:
Ah, it looks like this build tag was already present but a recent Go minor version changed the structure, so the diff is misleading.